src/store/reducers/peers.js

```diff
@@ -1,16 +1,21 @@
 import actionTypes from '../../constants/actions';
 
 /**
+ * The reducer for maintaining active peer
  *
- * @param {Array} state
- * @param {Object} action
+ * @param {Array} state - the current state object
+ * @param {Object} action - The action containing type and data
+ *
+ * @returns {Object} - Next state object
  */
 const peers = (state = {}, action) => {
   switch (action.type) {
     case actionTypes.activePeerSet:
-      return Object.assign({}, state, { active: action.data });
+      return Object.assign({}, state, { data: action.data });
```

Comment:
In this case, data makes perfect sense. There might be tens of peers available. but we just need to keep the tracks of the active one. so, I'm storing the data I get from lisk-js as its `data` and the potential configs we might add in lifecycle of Nano as `state`.